### DISCUSSION TITLE
Taking Orange Pi Ai Stick instead of Coral

### DISCUSSION THREAD

**Discussion by derrodi** [blakeblackshear/frigate#2165] [Ask A Question]
Since it is not possible to get an Coral i found on Ali the Orange Pi Ai Stick. Would it be possible to take this on instead of a coral usb stick?

**Comment by blakeblackshear** [comment]
Based on a quick review, no. It would need to support tensorflow lite similar to the way the Coral does.

**Reply by rpatel3001** [reply]
It appears that tensorflow is supported: https://www.reddit.com/r/OrangePI/comments/mywc0d/comment/gw09jrs/

I have basically no idea how ML works, is the different chipset incompatible because it expects the model parameters in some different format?

**Comment by ozett** [comment]
https://www.cnx-software.com/2019/10/24/orange-pi-ai-stick-lite-neural-compute-stick-free-training-tools/

some discussion on that page how to get started ...

**Reply by blakeblackshear** [reply]
Doesn't say anything about tensorflow lite support

**Reply by ozett** [reply]
seems also not fluidly or well maintained... maybe some hints in the linux-SDK, 
but Google-CORAL seems easier to maintain

**Comment by rpatel3001** [comment]
I'd love to use frigate, but it's pretty much impossible to get a reasonably priced coral, and I'm not sure how long it'll be until they're widely available again. If this seems like something that's possible but not a priority for the devs, I could buy a stick and hack away at it, but I'm not familiar with this codebase or ML/tensorflow at all. If it seems impossible or too difficult to maintain, no worries.

**Comment by ozett** [comment]
try these?
![image](https://user-images.githubusercontent.com/5937811/143928140-6ae9c5a4-fc66-4798-b073-a2fbbf81f215.png)

seems that the cheaper ones are mostly everywhere out of stock, (i just recently bought two at seeeds, but now out of stock)
i found only a german reseller with the cheaper ones out of stock, but the dual still sells...
![image](https://user-images.githubusercontent.com/5937811/143928678-189ac2d9-960d-4a58-81e7-0c5361ab8c18.png)

**Reply by rpatel3001** [reply]
Pollin won't ship it to the US. Seeed allowed me to order a USB device on backorder but I doubt it'll go through.

**Reply by rpatel3001** [reply]
Snagged a dual m.2 card off rsdelivers.com, hopefully that works.
